upload-to-sonatype-hadoop1.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "CLEANING"
+./gradlew clean
+
+for P in :gobblin-admin :gobblin-api :gobblin-azkaban :gobblin-compaction :gobblin-config-management:gobblin-config-core :gobblin-config-management:gobblin-config-client :gobblin-core :gobblin-data-management :gobblin-distribution :gobblin-example :gobblin-metastore :gobblin-metrics :gobblin-oozie :gobblin-rest-service:gobblin-rest-api :gobblin-rest-service:gobblin-rest-client :gobblin-rest-service:gobblin-rest-server :gobblin-runtime :gobblin-salesforce :gobblin-scheduler :gobblin-test :gobblin-test-harness :gobblin-utility :gobblin-yarn ; do 
+    echo "----------------------------" 
+    read -p "UPLOAD $P (y/n)" ans 
```

Comment:
No, this was meant:
(1) Give opportunity to check that there are no split sessions
(2) Skip packages that have already been uploaded 
(3) Allow to release individual packages.

(3) is not strictly necessary. It is possible to let packages accumulate in the same session and do a single release at the end.